### PR TITLE
chore(gen): merge branch 'master' into canary

### PR DESCRIPTION
**Pull request by kingcody**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-* text=auto
+* text eol=lf
```
Comment (kingcody) [inline .gitattributes]: This wasn't part of 552bf993592c42f99ec0f3f6afcf4ef1906c51a6, but I went ahead and included it as part of the merge.

File: .travis.yml [hunk 2]
```diff
@@ -17,3 +17,10 @@ cache:
     - node_modules
     - test/fixtures/node_modules
     - test/fixtures/bower_components
+notifications:
+  webhooks:
+    urls:
+      - secure: "DhPNqHXuUIeIGE9Ek3+63qhco+4MozXqMZL6dAKoq1MHQ2RAPO6SYIkUYZqDnuWYlwWao2EnTYcDREivIV/m/RnkP9bKlpX/n/RNJe+X4bwFaCU55fVKgkAFn3takSBC5SVoeTWHdWu3WhhqSdioWjT7mlE1wtt/RanSMb5Id8M="
```
Comment (kingcody) [inline .travis.yml]: Encrypted the webhook with the project's travis pubkey. This should prevent fork activity from being posted to the Gitter room.

File: app/templates/server/config/express.js [hunk 3]
```diff
@@ -39,7 +39,10 @@ module.exports = function(app) {
     secret: config.secrets.session,
     resave: true,
     saveUninitialized: true<% if (filters.mongoose) { %>,
-    store: new mongoStore({ mongoose_connection: mongoose.connection })<% } %>
+    store: new mongoStore({
+      mongooseConnection: mongoose.connection,
+      db: '<%= _.slugify(_.humanize(appname)) %>'
+    })<% } %>
```
Comment (kingcody) [inline app/templates/server/config/express.js]: We need to support some form of sequelize store.

File: .travis.yml (hunk 2 above)
Comment (kingcody) [inline .travis.yml]: Note, this does prevent PR's from forked repos from pushing activity as well; that may need to be considered.

**Comment by Awk34** [comment]
What's your opinion on the webhook? And want to make a task for the sequelize store?

**Comment by kingcody** [comment]
As far as the webhook goes, I think it would be cleaner and potentially less confusing to users for only the activity from this repo to be posted in the gitter chat. For instance I noticed several branches that I was testing on got published there.

**Pull request merged**